frontend/src/components/FederationTable/index.tsx

```diff
@@ -76,7 +77,8 @@ const FederationTable = ({
           >
             <Grid item>
               <RobotAvatar
-                shortAlias={params.row.shortAlias}
+                shortAlias={federated ? params.row.shortAlias : undefined}
+                hashId={federated ? undefined : params.row.shortAlias}
```

Comment:
NIT: for external coordinators we could use their `mainnet.url` instead of the `shortAlias` for the hashId. This way we ensure that the avatar generated matches the URL (permanent) instead of the alias (user dependentant) therefore the avatar will always be the same for everyone who adds this coordinator.